.env.example

```diff
@@ -92,8 +92,12 @@ DISCORD_CHANNEL_ID=
 DISCORD_SERVICE_ENABLED=false
 DISCORD_TOKEN="example-token"
 
-#DCL Data
+# DCL Data
 GATSBY_DCL_DATA_API=https://data.decentraland.vote/
 
 # Rollbar Token
 DAO_ROLLBAR_TOKEN=""
+
+# Newsletter
+BEEHIIV_API_KEY=
```

Comment:
we need this in definitions as a secret right?

Reply:
Yessss